src/widgets/search-box/search-box.js

```diff
@@ -37,6 +37,10 @@ const KEY_SUPPRESS = 8;
  * @return {Object}
  */
 
+const defaultQueryHook = function (query, setQueryAndSearch) {
+  setQueryAndSearch(query);
+};
+
```

Comment:
We use else where named functions, I think should try to enforce to use them when we do not need arrow functions:

```js
function defaultQueryHook(query, setQueryAndSearch) {
 ...
}
```